components/GlobalStyle/sub-components/Typography.jsx

```diff
@@ -90,9 +90,17 @@ ul, ol {
 const Typography = () => (
   <>
     <Style />
+    <link
+      crossOrigin="anonymous"
+      rel="preload"
+      href="https://fonts.googleapis.com/css?family=Nunito+Sans:400,600,700,400i,600i,700i&display=swap"
+      as="font"
+      type="font/woff2"
```

Comment:
You point out that the style is a font, but the href points to a css. This is divergent. I believe you are relying too much on the referral links you sent in slack. Note that in the mentioned links the problem they want to solve is importing a font and not importing a css

Reply:
```suggestion
      type="font/woff2"
```
That's why I added this format, with its presence the font extension is inside the css file, understand?

Reply:
I evaluated that when we use "as=Font" there is no need to use a typing